cross/boost/Makefile

```
@@ -1,11 +1,11 @@
 PKG_NAME = boost
-PKG_VERS = 1.66.0
+PKG_VERS = 1.76.0
 PKG_EXT = tar.bz2
 PKG_DIST_NAME = $(PKG_NAME)_$(subst .,_,$(PKG_VERS)).$(PKG_EXT)
 PKG_DIST_SITE = https://boostorg.jfrog.io/artifactory/main/release/$(PKG_VERS)/source
 PKG_DIR = $(PKG_NAME)_$(subst .,_,$(PKG_VERS))
 
-DEPENDS = cross/bzip2 cross/zlib
+DEPENDS = cross/bzip2 cross/zlib cross/python3
```

Until now we have support to build boost with python2 or python3 binding.
And we have support to define what boost libraries have to be built.
We can build selective boost libraries. If a package needs the boost libraries for python, the package itself has to define cross/python2 or cross/python3 or cross/python38 as dependency before the dependency of cross/boost.

```suggestion
DEPENDS = cross/bzip2 cross/zlib
```